Library/Homebrew/formulary.rb

```diff
@@ -47,6 +47,7 @@ def self.clear_cache
     super
   end
 
+  using PathnameWriteMkpath
```

Comment:
Could consider defining this either in here or in a dedicated class (rather than `extend/pathname`?)